crates/kitsune_p2p/types/Cargo.toml

```diff
@@ -23,8 +23,10 @@ serde = { version = "1", features = [ "derive", "rc" ] }
 serde_json = { version = "1", features = [ "preserve_order" ] }
 sysinfo = "0.15.9"
 thiserror = "1.0.22"
-tokio = { version = "0.2", features = [ "full" ] }
+tokio = { version = "1.3", features = [ "full" ] }
+tokio-stream = { version = "0.1", features = [ "sync", "net" ] }
 url2 = "0.0.6"
+quinn-proto = "0.7"
```

Comment:
This crate is supposed to be transport agnostic so apps that don't want to even include quic logic don't need to.

Can you utilize the `TransportError::other` function to make generic the quinn errors instead?

Reply:
Yes we can. By coercing the error into the `other` type we do lose the original type so a consumer of `transport_quic` - which I understand is _not_ transport agnostic - can't match on the `quinn` error type. I'm willing to accept that compromise for being able to remove this dependency :+1:

Reply:
Correct, but matching on the *quinn* type is not correct either - we might switch quic libraries at some point in the future. If there is some error type that may need to be branched upon, we should promote it to a kitsune variant that is not defined in a dependency.